### ISSUE TITLE
Chocolatey package

### ISSUE THREAD

**Issue by P6g9YHK6** [simple-acme/simple-acme#17]
We are currently using win-acme from the official Chocolatey repository for multiple reasons.

If we were to migrate to simple-acme in the future, it would be essential to have an equivalent package available.

The number of downloads from the Chocolatey repository alone demonstrates the demand and justifies the need for such an equivalent, as it would be crucial for many users.

And an automated migration would be appriciated with that :)
![image](https://github.com/user-attachments/assets/fccf511f-850b-4ca0-b992-32864c3a4c03)

**Comment by WouterTinus** [comment]
Frankly, I was not even aware that someone was publishing win-acme on Chocolatey, nobody asked or told me about this. I don't know who is doing this and cannot guarantee that these builds are (and will always be) the same as those published on GitHub. This may be someone who's trying to be helpful, but please proceed with caution if you're using this.

That said, I'm working on a big upgrade of the build/publishing scripts to include code signing, so I might register my own simple-acme package soon.

**Comment by P6g9YHK6** [comment]
it's mostly automated and open source so i do not worry about the legitimacy of the package
https://github.com/TheCakeIsNaOH/chocolatey-packages/tree/master/win-acme

And on top there are automated/manual check done on the packages it does give some reassurance about the content of the packages
https://gist.github.com/choco-bot/be73a95713bea572dab9e18fc346ad20
![image](https://github.com/user-attachments/assets/d43ebbab-f591-4b2e-921a-6ae227b38c3c)
![image](https://github.com/user-attachments/assets/92bc3d51-3d26-4977-842c-482f20ab7283)

Personally, i trust more a package chocolatey than an a .exe coming directly from the dev's of a software due to these checks

not doing any advertising i am just a tech that is happy that running once a month `choco upgrade all` on my infra will keep everything up to date without worry

Anyway.

An officially supported package will be appreciated ☺️

**Comment by webprofusion-chrisc** [comment]
@P6g9YHK6 anyone can upload a package version that deletes all your documents, it doesn't have to have a virus to do that. You are welcome to trust particular publishers but being published on chocolatey does not make something trustworthy.

@WouterTinus pretty sure you can get them to redirect the entry to your official package if you publish one. I'd suggest starting to include helpful guidance to existing users that they should migrate to simple-acme when they launch win-acme as not many will find out on their own.

**Comment by P6g9YHK6** [comment]
> @P6g9YHK6 anyone can upload a package version that deletes all your documents, it doesn't have to have a virus to do that. You are welcome to trust particular publishers but being published on chocolatey does not make something trustworthy.
> 
> @WouterTinus pretty sure you can get them to redirect the entry to your official package if you publish one. I'd suggest starting to include helpful guidance to existing users that they should migrate to simple-acme when they launch win-acme as not many will find out on their own.

I agree there is different risks associated but it does move some asspect of supply chains attack to a more "transparent" ground 


And I would never have found this project without trying to do a PR to win-acme and this makes me already the 0.1% of users so yea nobody will ever find out without so yea some documentation will probably be needed for a start on the website/github repo of win-acme

**Comment by WouterTinus** [comment]
@P6g9YHK6 - I understand the added value of package managers, but I'm just letting you and others that might stumble on this thread via Google know that the currently available win-acme package on Chocolatey is not an "official" one - and to take that into consideration before you use it. 

As stated before, I will look into creating official packages for simple-acme once I have code signing fully enabled, so that users of Chocolatey, win-get or whatever other third party distribution channel can be sure that they're getting original, unmodified builds straight from the public source.

Marketing for simple-acme will also pick up after that's all been sorted :)

**Comment by geralt18** [comment]
@WouterTinus

**Comment by WouterTinus** [comment]
Hi @geralt18, the package is currently awaiting moderation and can only be found via the direct link: https://community.chocolatey.org/packages/simple-acme/2.3.1.9

Also, I haven't gotten around to publishing the plugins yet. There being ~25 of them, this requires some automation.

**Comment by P6g9YHK6** [comment]
> Hi [@geralt18](https://github.com/geralt18), the package is currently awaiting moderation and can only be found via the direct link: https://community.chocolatey.org/packages/simple-acme/2.3.1.9
> 
> Also, I haven't gotten around to publishing the plugins yet. There being ~25 of them, this requires some automation.

Good job thanks!

Also did you saw the AV's postive match on chocolatey? Not a problem but just wanted to let you know.

**Comment by WouterTinus** [comment]
Yes I've seen that, as far as i can tell these are a couple of false positives in some relatively unknown AV products, i don't believe there's anything I can do about it. Chocolaty luckily doesn't consider it a huge problem unless more than 5 AVs flag the package.

**Comment by P6g9YHK6** [comment]
there was a rejection on the icons

**Comment by WouterTinus** [comment]
I think that was just on the trial submission that I've made. Currently waiting for this one to get approved: https://community.chocolatey.org/packages/simple-acme/2.3.1.9

I hope this wait is not representative, almost a month delay is pretty long if you're waiting for a specific fix that you already know is available from the source.

**Comment by P6g9YHK6** [comment]
This is probably just the 1st time 
Automated package are generally available within 24h of release

**Comment by WouterTinus** [comment]
The main package was approved, now just waiting for the plugins 👍

**Comment by P6g9YHK6** [comment]
is there some documentation somewhere on how to upgrade from win-acme to simple-acme ?

**Comment by WouterTinus** [comment]
Just extract simple-acme to your old win-acme folder and it should work just fine. 

If your were using win-acme via Chocolately you will have to move the configuration files in %programdata%\win-acme to %programdata%\simple-acme. Alternatively you can update `settings.json` in C:\tools\simple-acme to change the "clientname" property from "simple-acme" to "win-acme" so that it will look in the old folders.

**Comment by P6g9YHK6** [comment]
copy 
%programdata%\win-acme -> %programdata%\simple-acme
install simple-acme
check that everything is fine 
delete %programdata%\win-acme

seems fine enough to me :) thanks !
